Sources/MockoloFramework/Templates/ThrowingKindTemplate.swift

```diff
@@ -15,13 +15,26 @@
 //
 
 extension ThrowingKind {
-    func applyThrowingTemplate() -> String? {
+    /// if this method is called to render handler for closure, rethrows should be replaced with throws.
+    ///
+    /// - example:
+    ///     ```
+    ///     // handler shouldn't use rethrows, instead use throws
+    ///     var fooHandler: (() -> throws -> Void) throws -> Void
+    ///     func foo(bar: () throws -> Void) rethrows -> Void
+    ///     ```
+    func applyThrowingTemplate(
+        appliesforClosureHandler: Bool
+    ) -> String {
         switch self {
         case .none:
-            return nil
```

Comment:
This change may cause effects to trivia. Sadly, it is not covered in tests.

Reply:
Thank you! I am not sure the concrete test case but fixed in 733771e .
I also guess it depends on how caller of this method process this return value (I don't understand failed case well).

Reply:
Yes, it's a problem of the use site. You can fix either one, but there's no need to change.